### PR TITLE
P4C: Convert expandables to tables

### PR DESCRIPTION
**Pull request by virginiacc**
Convert expandables to tables on grad path tool review screen to improve usability.

## Changes

- Convert expandables to tables on review screen

## How to test this PR

1. Check out branch and run gulp
2.  Navigate to `Review your financing plan` screen in grad path tool
3. Verify data output in tables

## Screenshots

localhost_8000_paying-for-college_your-financial-path-to-graduation__iped=110635 pid=4510-3 houp=onCampus typp=bachelors lenp=4 ratp=inState depp=dependent houx=1395 fdx=498 clhx=127 trnx=750 hltx=265 entx=185 retx=282 taxx=143 othx=328

## Checklist

- [ ] PR has an informative and human-readable title
  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [ ] Future todos are captured in comments and/or tickets
- [ ] Project documentation has been updated, potentially one or more of:
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

#### Browser testing

Visually tested in the following supported browsers:
- [ ] Firefox
- [ ] Chrome
- [ ] Safari
- [ ] Edge 18 (the last Edge prior to it switching to Chromium)
- [ ] Internet Explorer 11 and 8 (via emulation in 11's dev tools)
- [ ] Safari on iOS
- [ ] Chrome on Android

#### Accessibility

- [ ] Keyboard friendly (navigable with tab, space, enter, arrow keys, etc.)
- [ ] Screen reader friendly
- [ ] Does not introduce new errors or warnings in [WAVE](https://wave.webaim.org/extension/)

#### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Does not introduce new lint warnings
- [ ] Flexible from small to large screens

### CODE REVIEW

**Review by mistergone — APPROVED** [review]

File: cfgov/unprocessed/apps/paying-for-college/css/college-costs/college-costs.less [hunk 1]
```diff
@@ -97,8 +97,20 @@
 			}
 
 			.financial-item + .financial-item {
-				margin-top: unit( @grid_gutter-width / 3 / @base-font-size-px, em );;
+				margin-top: unit( @grid_gutter-width / 3 / @base-font-size-px, em );
 			}
+
+			dl {
+				dt {
```
Comment (anselmbradford) [inline cfgov/unprocessed/apps/paying-for-college/css/college-costs/college-costs.less:104]: What's going on with the closing `}` here? Is the indenting just off for the parent `dl`?

File: cfgov/unprocessed/apps/paying-for-college/css/college-costs/college-costs.less [hunk 2]
```diff
@@ -97,8 +97,20 @@
...
+			        font-size: inherit;
+			    	text-transform: none;
+			    	letter-spacing: normal;
+			    	margin-bottom: 0;
+		    	}
```
Comment (anselmbradford) [inline cfgov/unprocessed/apps/paying-for-college/css/college-costs/college-costs.less:109]: What's going on with the closing `}` for the `dt` here? Is the indenting just off for the parent `dl`?

File: cfgov/unprocessed/apps/paying-for-college/css/college-costs/college-costs.less [hunk 3]
```diff
@@ -167,6 +179,12 @@
     	list-style: none;
 	}
 
-
-	
+    .o-table__financial {
+    	.indented-row td:first-child {
+        	padding-left: unit( @grid_gutter-width * 2/3 / @base-font-size-px, em );
```
Comment (anselmbradford) [inline cfgov/unprocessed/apps/paying-for-college/css/college-costs/college-costs.less:184]: `padding-left: unit( @grid_gutter-width / 1.5 / @base-font-size-px, em );` might be clearer?

File: cfgov/unprocessed/apps/paying-for-college/css/college-costs/college-costs.less [hunk 4]
```diff
@@ -167,6 +179,12 @@
...
+    	}
+    	.subtotal-row {
+        	font-weight: 600;
+    	}
+    }
 }
```
Comment (anselmbradford) [inline cfgov/unprocessed/apps/paying-for-college/css/college-costs/college-costs.less:190]: Could throw the trailing newline in while you're here.

File: cfgov/unprocessed/apps/paying-for-college/css/college-costs/college-costs.less (hunk 2 above)
Comment (virginiacc) [reply cfgov/unprocessed/apps/paying-for-college/css/college-costs/college-costs.less:109]: I think this is just an indentation issue -- I'll push a commit that fixes the indentation in the file.

**Pull request merged**